analyzers/tests/SonarAnalyzer.TestFramework/Verification/CodeFixVerifier.cs

```diff
@@ -82,8 +81,8 @@ public void VerifyFixAllProvider(FixAllProvider fixAllProvider, ParseOptions par
             .AssertExpected(pathToExpected, $"{nameof(VerifyFixAllProvider)} runs {fixAllProvider.GetType().Name} once");
     }
 
-    private string RetrieveCodeFixTitle(CodeFixProvider codeFix, State state) =>
-        state.Diagnostics.SelectMany(diagnostic => ActionToApply(codeFix, state.Document, diagnostic)).FirstOrDefault()?.Title;
+    private string CodeFixTitle(CodeFixProvider codeFix, State state) =>
+        state.Diagnostics.SelectMany(x => ActionToApply(codeFix, state.Document, x)).First().Title;
```

Comment:
It will never reach `OrDefault` part. Such case would already fail on `state.Diagnostics.Should().NotBeEmpty();` before entering here.